src/main/resources/modid.mixins.json

```diff
@@ -4,9 +4,11 @@
   "package": "net.fabricmc.example.mixin",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
+    "ExampleMixin"
```

Comment:
The Mixin's target `TitleScreen` only exists on the client, so the mixin will fail to apply

Reply:
Oh, my bad. 
I think it might be more useful to have a non-client-only mixin exmaple, though. 
I may create another PR that just adds a second example (I'll make sure it functions first).

Reply:
Something simple at most would probably be needed.